sportnet/sportnet/playgrounds/models.py

```diff
@@ -4,14 +4,24 @@
 
 
 class PlayGround(TimeStampedModel):
+
+    title_action = models.CharField(max_length=128, verbose_name='Название мероприятия', blank=False, null=False)
+    admins = models.ManyToManyField(get_user_model(), verbose_name='Администратор',
+                                    verbose_name_plural='Администраторы', related_name='own_playgrounds')
+    players = models.ManyToManyField(get_user_model(), verbose_name='Участники', related_name='in_playgrounds')
+    quantity_players = models.PositiveIntegerField(verbose_name='Количество участников', default=0)
```

Comment:
На случай, если не регламентируется, чтобы там "0" глаза не мозолил
```suggestion
    quantity_players = models.PositiveIntegerField(verbose_name='Количество участников', blank=True, null=True)
```